org.blender.Blender.appdata.xml

```diff
@@ -0,0 +1,121 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+    <id>org.blender.BlenderLTS</id>
+    <launchable type="desktop-id">org.blender.Blender.desktop</launchable>
+    <name>Blender</name>
+    <summary>Free and open source 3D creation suite</summary>
+    <metadata_license>CC0-1.0</metadata_license>
+    <project_license>GPL-3.0</project_license>
+    <description>
```

Comment:
we can't have blender and the lts version looking otherwise identical - we'll need to make some changes here to explain what the benefit/cost of the LTS version is.

Reply:
What kind of changes do you have in mind ? 😕

Reply:
Maybe saying Blender LTS in the title? And another paragraph in the description that has something like:

> The Blender LTS program is aimed at ensuring that long-lasting projects can be executed using a stable Blender version, which will provide critical fixes throughout a 2-year time span. The LTS version will not have any new features, API changes or improvements.

Which is the native description from https://www.blender.org/download/lts/